test/evergreen.yml

```
@@ -890,8 +889,8 @@ functions:
           echo "Using num_jobs '-j ${num_jobs}' for 'unit test'"
           threads_command="-j ${num_jobs}"
         fi
-        if [ ${check_coverage|false} = true ]; then
-            ${python_binary|python3} ../test/suite/run.py ${ignore_stdout} ${unit_test_args|-v 2} ${unit_test_variant_args} $threads_command $(cat $do_run) 2>&1 || echo "Ignoring failed test as we are checking test coverage"
+        if [ ${ingore_failures|false} = true ]; then
```

Looks like there is a typo:

```suggestion
        if [ ${ignore_failures|false} = true ]; then
```